File: UITableViewThreeEventsDemo/Demo-objc/Demo-objc/Cell.m
//
//  Cell.m
//  Demo-20160419
//
//  Created by crw on 4/19/16.
//  Copyright © 2016 crw. All rights reserved.
//

#import "Cell.h"

@implementation Cell

- (void)setModel:(Model *)model{
    _model = model;
    
    self.titleLabel.text = model.title;
}

@end

File: UITableViewThreeEventsDemo/Demo-swift/Demo-swift/Cell.swift
//
//  Cell.swift
//  Demo-20160418
//
//  Created by crw on 4/18/16.
//  Copyright © 2016 crw. All rights reserved.
//

import UIKit

class Cell: UITableViewCell {

    @IBOutlet weak var titleLabel: UILabel!
    
    @IBOutlet weak var delBtn: UIButton!
    
    var model : Model? {
        didSet{
            if let model = model {
                titleLabel.text = model.title
            }
        }
    }
    
}
